packages/sdk/src/ClientSide.ts

```diff
@@ -54,29 +57,43 @@ export class ClientSide {
 		payload.customer = payload.customer ?? {}
 		payload.customer.source_id = payload.customer.source_id ?? this.trackingId
 
-		return this.client.post<$FixMe>('/redeem', payload, { code })
+		return this.client.post<T.ClientSideRedeemResponse>('/redeem', payload, { code })
 	}
-	public publish(campaign: string, payload: $FixMe = {}) {
+	// @todo - add payload Type
+	public publish(campaign: string, payload: T.ClientSidePublishPayload = {}) {
 		assert(isString(campaign), 'client.publish - campaign is required to publish a voucher')
-		assert(isObject(payload), 'client.redeem - expected payload to be an object')
+		assert(isObject(payload), 'client.publish - expected payload to be an object')
 
 		payload.customer = payload.customer ?? {}
 		payload.customer.source_id = payload.customer.source_id ?? this.trackingId
 		payload.channel = payload.channel ?? 'Voucherify.js' // @todo - removed hard-coded channel
 
-		return this.client.post<$FixMe>('/publish', payload, { campaign })
+		// @todo - add ReturnType
+		return this.client.post<T.ClientSidePublishResponse>('/publish', payload, { campaign })
```

Comment:
Why the `@todo` comment? Isn't `T.ClientSidePublishReponse` the return type? 😅

Reply:
Sorry, fixing it.